### DISCUSSION TITLE
Database User Security

### DISCUSSION THREAD

**Discussion by savannt** [Budibase/budibase#8178] [Help]
Hey I am curious if its possible to create user-based tables... at the moment all database tables can be queried so as long as the user has valid budibase auth cookies.

**Comment by melohagan** [comment]
Hey @B3N909 

The best you can do at the moment is assign app specific roles to users: https://docs.budibase.com/docs/user-roles#app-specific-roles

Then you can manage access to tables based on role: https://docs.budibase.com/docs/attachments#manage-access
